doc/cli.markdown

```diff
@@ -950,7 +951,7 @@ Examples:
 	balena ssh 192.168.0.1 --verbose
 	balena ssh f49cefd.local my-service
 
-Warning: 'balena ssh' requires an openssh-compatible client to be correctly
+Warning: `balena ssh` requires an openssh-compatible client to be correctly
```

Comment:
`cli.markdown` is auto generated, so these changes would have to go in, I think, `lib/actions/ssh.ts`, then run `npm run build`.

Reply:
noted, thanks